session.rb

```diff
@@ -0,0 +1,45 @@
+require_relative "track" 
+
+class Session
+  attr_reader :talks, :year, :month, :day
+
+  def initialize(talks)
+    @talks = talks
+    @year, @month, @day = 2020, 8, 3
+    @track = nil
+    @available_minutes = nil
+    @talk_start_time = nil
+  end
+
+  def morning_sessions(track)
+    @start_time = Time.local(year, month, day, 9)
+    @end_time = Time.local(year, month, day, 12)
+    common_job(track, @start_time, @end_time)
+  end
+
+  def afternoon_sessions(track)
+    @start_time = Time.local(year, month, day, 13)
+    @end_time = Time.local(year, month, day, 17)
+    common_job(track, @start_time, @end_time)
+  end
+
+  private
+
+  def common_job(track, start_time, end_time)
```

Comment:
Don't use names like common_job. That's not at all helpful and tells me absolutely nothing about what the method does.